install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -11,144 +11,406 @@
+link:../../admin_guide/manage_scc.html[Managing Security Context Constraints (SCC)]
+and
+link:../../architecture/additional_concepts/authorization.html#security-context-constraints[SCC concepts].
+
+As a quick background, the Openshift
+link:../../architecture/additional_concepts/storage.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage and gives users
+a way to request those resources without requiring knowledge of the underlying infrastructure.
+Persistent volumes (PV) are global resources and not tied to a specific project (namespace). 
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
+volume claims (PVC)] are local to the project, and multiple PVCs within the same project
+can bind to the same PV. However, once a PVC binds to a PV, that PV cannot be bound by a claim
+outside of the first claim's namespace. If the underlying storage needs to be accessed by
+multiple projects then each project needs its own PV, which can point to the same physical
+storage. In this sense, a bound PV is tied to a namespace. For a detailed PV and PVC example,
+see the guide for
+https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and MySQL using NFS].
+
+*NOTE:*
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+
+Accessing persistent storage requires coordination between the cluster (and/or storage)
+administrator and the end-developer. The cluster admin creates persistent volumes (PVs),
+which abstract the underlying physical storage. The developer creates pods and,
+optionally, persistent volume claims (PVCs), which bind to PVs, based on matching
+criteria, such as capacity. For the admin, granting pods access to PVs involves knowing
+the group ids and/or user id assigned to the actual storage, selinux considerations, and,
+finally, ensuring that these ids are allowed in the range of legal ids defined for the
+project and/or the SCC that matches the requirements of the pod.
+
+Group ids, the user id, and selinux values are defined in the `*SecurityContext*` stanza
+in a pod definition. Group ids are global to the pod and apply to all containers defined
+in the pod. User ids can also be global, or specific to each container. Four stanzas
+control access to volumes:
+link:#fsgroup[`fsGroup`],
+link:#supplemental-groups[`supplementalGroups`],
+link:#user-id[`runAsUser`], and
+link:#selinux[`SELinuxOptions`].
+
+[[scc]]
+==== SCCs, Defaults, and Allowed Ranges
+SCCs influence whether or not a pod is given a default user id, fsgroup id, supplemental
+group id, and selinux label; and, whether or not ids supplied in the pod spec (or in the
+image) will be validated against a range of allowable ids. If validation is required and
+fails then the pod will also fail.
+
+SCCs define strategies, such as `RunAsUser`, `supplementalGroups`, and `fsGroup`. These
```

Comment:
They are all SCC stragegies, aren't they? What difference in these strategies is important to call out or re-categorize here?